spotseeker_server/management/commands/techloan/spotseeker.py

```diff
@@ -251,6 +256,12 @@ def upload_data(self):
                         )
                         continue
 
+                image = self._download_image(image_url, cte_type_id) \
+                    if image_url and (recent_changes or not image_exists) \
```

Comment:
The logic was changed from recent_changes to recent_changes or not image_exists so sync_techloan will keep trying to download the image if an item has an image_url but no images every time sync_techloan is run. This also lets the test server still download the bh.static.photo images even though they are marked as not having recent changes.